src/stream_manager.rs

```diff
@@ -229,6 +231,77 @@ impl StreamManager {
         }
     }
 
+    ///
+    /// list scope
+    ///
+    #[pyo3(text_signature = "($self)")]
+    pub fn list_scope<'p>(&self, _py: Python<'p>) -> PyResult<Vec<String>> {
+        let controller = self.cf.controller_client();
+        let scope_result = list_scopes(controller);
+        futures::pin_mut!(scope_result);
+
+        let mut scope_vector = Vec::new();
+        // Used tokio::runtime::Runtime to block on the async code
+        let _result = tokio::runtime::Runtime::new().unwrap().block_on(async {
```

Comment:
Don't name it with an _ as it is not unused

Reply:
Removed the _ from result variable